convert_two_1d_arrays_into_2d_array.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+
+def convert_two_1_d_arrays_into_2_d_array(arr1, arr2):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/